src/data/nat/basic.lean

```diff
@@ -401,11 +415,30 @@ lt_of_mul_lt_mul_left
     ... < n * k : h)
   (nat.zero_le n)
 
+lemma le_mul_of_div_lt {a b c : ℕ} (h : a / c < b) (w : 0 < c) : a ≤ b * c :=
```

Comment:
why is this asymmetric? We have lt in the assumption and le in the goal. Surely one or the other can be strengthened so they are the same?

Reply:
Yes, of course, we can strengthen the conclusion (but not weaken the hypothesis).